generate-sales-report/README.md

```diff
@@ -0,0 +1,116 @@
+# Generate custom sales reports automatically
+
+_Bring sales data from your data warehouse directly into
+ Google Sheets, and then automatically generate presentations 
+ to summarize wins and opportunities for particular regions 
+ and customers._
+
+Last updated: November, 2019
+
+Annual and quarterly reports are a standard part of business 
+for many enterprises. But automation can help eliminate the 
+repetition inherent in these periodic obligations. 
+If you find yourself needing to analyze and present findings 
+from large sets of stored data, consider using automation 
+to help streamline your reporting. 
+
+This solution creates a tool that connects to sales data in 
+BigQuery, an analytics data warehouse, from directly within 
+Google Sheets. A configuration sheet allows a user to provide 
+parameters for the report, such as the Account Name and 
+Region. With the click of a button, a customized report
+with the latest sales data is automatically created in just 
+a matter of seconds! 
+
+
+![summary](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/generate-sales-report/demo.gif)
+
+Note: This solution requires a G Suite Business, Enterprise, 
+or Education account, and a Google Cloud Platform account and 
+project.
+
+## Technology highlights
+
+- Uses the [Sheets data connector for BigQuery][connector-support]
+  to access tables in a data warehouse from directly 
+  within Google Sheets.
+- Uses [Google Slides][google-slides] to create a templatized 
+  report.
+- Uses [Apps Script][apps-script] to 
+  create a chart in Google Sheets, and merge it, along with 
+  sales data, into the template report.
+
+## Try it
+
+### Set up a Google Cloud Platform project
+This solution requires a [Google Cloud Platform account and project][gcp-account]. The service used in this solution, BigQuery, 
+has a [sandbox environment][sandbox] 
+that you can use to test this solution.
+
+1. [Sign in][sign-in] with your G Suite Business, Enterprise, 
+or Education Account credentials. 
+1. In the [GCP Console][console], select or create a new GCP project.
+
+
+### Create a templatized slide deck
+
+1. Make a copy of the template slide deck [here][slide-deck].
+1. Identify the unique ID of your Slides document. The ID can 
+be derived from the URL: `https://docs.google.com/presentation/d/`_**`slideId`**_`/edit`
+
+### Set up the configuration spreadsheet
+
+1. Make of copy of the template spreadsheet [here][spreadsheet].
+1. From the spreadsheet, open the script editor by selecting 
+**Tools > Script editor**.
+1. Copy and paste your Slides document ID into line 1 of 
+`Constants.gs` replacing `YOUR_SLIDES_ID` and maintaining 
+the quotes.
+1. Save the changes by navigating to **File > Save**.
+
+
+### Generate a new report
+
+1. Navigate to the Generator tab of your spreadsheet.
+1. Choose an Account Name and Region in drop-down cells.
+1. Navigate to the Data Results tab and click **Refresh** 
```

Comment:
Can we add a screenshot here? We can probably crop & reuse the images from the blog post

Reply:
These steps are what is shown in the gif that I have in the intro section...do you want a screenshot of one of the particular steps?

Reply:
Gif is there!